code-studio/src/js/pd/workshop_dashboard/attendance/workshop_attendance.jsx

```diff
@@ -0,0 +1,194 @@
+/* global React */
+
+var SessionTime = require('../components/session_time.jsx');
+var SessionAttendance = require('./session_attendance.jsx');
+var Row = require('react-bootstrap').Row;
+var ButtonToolbar = require('react-bootstrap').ButtonToolbar;
+var Button = require('react-bootstrap').Button;
+var Tabs = require('react-bootstrap').Tabs;
+var Tab = require('react-bootstrap').Tab;
+var Col = require('react-bootstrap').Col;
+
+var WorkshopAttendance = React.createClass({
+  contextTypes: {
+    router: React.PropTypes.object.isRequired
+  },
+
+  propTypes: {
+    params: React.PropTypes.shape({
+      workshopId: React.PropTypes.string.isRequired,
+      sessionIndex: React.PropTypes.string
+    }).isRequired
+  },
+
+  getInitialState: function () {
+    return {
+      loading: true
+    };
+  },
+
+  componentDidMount: function () {
+    // Response format:
+    // [
+    //   state: _workshop state_,
+    //   session: {id. start, end},
+    //   attendances: [{name, email, enrolled, user_id, in_section, attended}]
+    // ]
+    this.loadRequest = $.ajax({
+      method: "GET",
+      url: `/api/v1/pd/workshops/${this.props.params.workshopId}/attendance`,
+      dataType: "json"
+    }).done(function (data) {
+      this.setState({
+        loading: false,
+        workshopState: data.state,
+        sessionAttendances: data.session_attendances,
+        adminActions: data.admin_actions
+      });
+    }.bind(this));
```

Comment:
Ideally yes. I guess we could display an error message or something. Maybe good to add in the post-facilitator-summit improvements.